usecase/implementation/src/main/java/eu/openg/guessnumberapi/usecase/implementation/GetGamesInteractor.java

```diff
@@ -0,0 +1,28 @@
+package eu.openg.guessnumberapi.usecase.implementation;
+
+import eu.openg.guessnumberapi.domain.Game;
+import eu.openg.guessnumberapi.gateway.api.GameRepository;
+import eu.openg.guessnumberapi.usecase.api.BoundaryGame;
+import eu.openg.guessnumberapi.usecase.api.GetGamesUseCase;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GetGamesInteractor implements GetGamesUseCase {
+    private final GameRepository gameRepository;
+
+    public GetGamesInteractor(GameRepository gameRepository) {
+        this.gameRepository = gameRepository;
+    }
+
+    @Override
+    public List<BoundaryGame> fetchGames() {
+        List<Game> games = gameRepository.fetchGames();
+        return convert(games);
```

Comment:
inline metodą ir returną

Reply:
siūlyčiau išsikelti į metodą tik šitą gabaliuką: new BoundaryGame(game.getGameId(), game.getGuessCount(), game.getActualNumber())